services/src/datasets/external/pangaea/mod.rs

```diff
@@ -420,17 +422,17 @@ mod tests {
 
         let meta: Box<
             dyn MetaData<OgrSourceDataset, VectorResultDescriptor, VectorQueryRectangle>,
-        > = provider.meta_data(&id).await.unwrap();
+        > = provider.meta_data(&id.clone().into()).await.unwrap();
```

Comment:
Is it possible to implement Borrow or AsRef here for the id?

Reply:
I don't think it is possible because it expects reference to something that owns the fields (of the `DataId`), but I could only create something that borrows the fields. But if you know a solution that would be nice.